packages/kbn-optimizer/limits.yml

```diff
@@ -107,7 +107,7 @@ pageLoadAssetSize:
   dataVisualizer: 27530
   banners: 17946
   mapsEms: 26072
-  timelines: 251886
+  timelines: 330000
```

Comment:
I don't like how big this is getting, but between https://github.com/elastic/kibana/issues/105941 and https://github.com/elastic/kibana/pull/105207#issuecomment-882714736 I understand this is just some debt we're taking on temporarily.

Reply:
Exactly, thanks